api/exclusions/exclusionService.ts

```diff
@@ -31,8 +31,15 @@ export async function confirmUserExclusion(req: EnhancedRequest, res: Response,
       userType: 'Judicial',
     },
   ];
-  if (['400, 401, 402, 403, 500, 503'].indexOf(req.body.exclusionDescription)) {
-    return res.status(req.body.exclusionDescription).send('error: {status: req.body.exclusionDescription}"');
+  if (req.body.exclusionDescription && ['400, 401, 402, 403, 500, 503'].indexOf(req.body.exclusionDescription)) {
+    return res.status(req.body.exclusionDescription).send(`error: {status: ${req.body.exclusionDescription}}`);
```

Comment:
I guess this will be using my version?

Reply:
I'll get your changes - might have to tweak tests a bit actually

Reply:
I should get your changes actually, yes